packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -1,30 +1,54 @@
 // @ts-nocheck
-import { ToastOptions } from 'react-native-toast-notifications/lib/typescript/toast'
-import React from 'react'
+import {
+  Pressable,
+  PressableStateCallbackType,
+  Text,
+  TextProps,
+  TextStyle,
+  View,
+  ViewProps,
+  ViewStyle,
+  useWindowDimensions,
+} from 'react-native'
+import { ToastProps } from 'react-native-toast-notifications/lib/typescript/toast'
+import { useTranslation } from 'react-i18next'
+import React, { FC } from 'react'
 import Toast, { ToastType } from 'react-native-toast-notifications'
 import ToastContainer from 'react-native-toast-notifications'
 
-type modifyToastOptions = Omit<ToastOptions, 'data'> & {
+import { Icon, IconProps } from '../Icon/Icon'
+import { Spacer } from '../Spacer/Spacer'
+import { useTheme } from '../../utils'
+
+// let snackbarOffset: number = 10
+
+type snackbarData = {
   data?: {
-    onActionPressed?: () => void
+    /** true if snackbar represents an error state */
     isError?: boolean
-    actionBtnText?: string
-    isUndo?: boolean
+    /** message text A11y label override */
+    messageA11y?: string
+    /** action button onPress logic for "Try again" (isError=true) or "Undo" button */
+    onActionPressed?: () => void
   }
 }
 
+/**
+ * Structured to allow modification of `react-native-toast-notifications` ToastOptions
+ * type, but Snackbar component locks down all non-data options presently
+ *
+ * In the future, this may change (e.g. allowing non-indefinite `duration`)
+ */
+type modifyToastOptions = snackbarData
```

Comment:
Will this be modified at all in the 3 follow-up of tickets? If not, I feel like it  should be removed until this future change is needed, but maybe I'm missing something.

Reply:
No, I don't expect this would change as part of MVP or currently planned post-MVP changes. 

I retained it because it was picking apart the library typing in a way that may not necessarily be intuitive when revisiting later. Maybe that's fine since it was only mentioned in passing the possibility of allowing controlling the duration or other aspects so we may never end up doing it, but I liked retaining the breadcrumb that the `data` prop is for our custom data while there are other Toast-library level values that could potentially be updated here outside `data` if we allow them in the future. It lingering also leaves a record of why it's called with the nested `data` object since it could be simplified, but then would result in breaking changes if we ever did shift to allowing other props here in the future.